### ISSUE TITLE
The 2.8.0 npm package contains code coverage code

### ISSUE THREAD

**Issue by rhinoduck** [reactiflux/discord-irc#535]
I am not a JS/Node/npm guy, so I hope I am not just missing something; but I would not expect that a version meant for distribution would contain what seems like autogenerated code coverage testing code.

It can be found in all of the following files:

```
discord-irc/dist/logger.js
discord-irc/dist/errors.js
discord-irc/dist/helpers.js
discord-irc/dist/bot.js
discord-irc/dist/validators.js
discord-irc/dist/index.js
discord-irc/dist/cli.js
discord-irc/dist/formatting.js
```

discord-irc was installed using the `npm install discord-irc` command (node.js12.16.1, npm 6.14.2).

EDIT: Removed a symlink from the list of files.

**Comment by Throne3d** [comment]
So it does! Seems like I misunderstood some things when creating PR #523. This should be fixed in #536, hopefully.

**Comment by rhinoduck** [comment]
Np. Thanks for taking care of it so quickly :)
